deploy/crds/zookeeper_v1beta1_zookeepercluster_crd.yaml

```diff
@@ -20,6 +20,14 @@ spec:
     type: integer
     description: The number of ZooKeeper servers in the ensemble that are in a Ready state
     JSONPath: .status.readyReplicas
+  - name: version
+    type: string
+    description: The current Zookeeper version
+    JSONPath: .status.currentVersion
+  - name: Desired version
+    type: string
+    description: The desired Zookeeper version
+    JSONPath: .status.targetVersion    
```

Comment:
for desired version u should pick the value from `image.tag`
since we would not want this value to be empty at any point

Reply:
Done.